### DISCUSSION TITLE
Отдельный пользователь Web интерфейса для прослушивания разговоров

### DISCUSSION THREAD

**Discussion by sergisx** [mikopbx/Core#73] [Ideas (Идеи)]
Думаю, было бы полезно пользователю авторизироваться под учёткой, например, своего extension number, чтобы иметь доступ к статистике звонков с возможностью прослушивать свои разговоры или иметь права смотреть и слушать звонки сотрудников, т.к. несекьюрно ему давать админ учётку

**Comment by jorikfon** [comment]
Такая задача есть в виде отдельного интерфейса для пользователя в самой PBX.
Также сейчас при использовании интеграции с 1С, это все доступно в внешней панели телефонии. У вас сейчас какая нибудь CRM используется в компании?


<img width="1093" alt="NBScreenShot 2021-02-16 at 12 35 16" src="https://user-images.githubusercontent.com/5922918/108044524-6ee48a80-7053-11eb-801e-0844b02c83d1.png">

**Reply by sergisx** [reply]
Вот начали сегодня внедрять Битрикс.
>> Такая задача есть в виде отдельного интерфейса для пользователя в самой PBX.
Т.е. уже реализовано или есть задача создать такой интерфейс? Извините, не понял

**Reply by jorikfon** [reply]
Это реализовано, пока нет раздела на сайте, пилим.  Наиболее близкое по описанию  функционала здесь https://telefon.miko.ru/products/panel-mango/
